### PR TITLE
Added some stuff for the ping event

### PR DESCRIPTION
**Pull request by oatmealraisin**
Woo time to figure out what happens with a PR

### CODE REVIEW

**Comment by oatmealraisin** [comment]
Comment?

**Comment by oatmealraisin** [comment]
Request a review!

File: listener.go [hunk 1]
```diff
@@ -14,22 +15,25 @@ var secretKey []byte
 
 func ReceiveWebhook(w http.ResponseWriter, req *http.Request) {
 	//	client := github.NewClient(nil)
+	secretKey = []byte("killme")
```
Comment (finiteautomatom) [inline listener.go:18]: What is thing? Hahaha

**Review by finiteautomatom — CHANGES_REQUESTED** [review]

**Comment by oatmealraisin** [comment]
Comment

**Pull request merged**